clpy/backend/opencl/utility.pyx

```diff
@@ -140,18 +144,19 @@ cdef RunNDRangeKernel(
     )
     api.WaitForEvents(1, &event[0])
 
-cdef __device_typeof_size():
+__typessof_sizes = ['ulong'] * env.num_devices
```

Comment:
My understanding is that `__typessof_sizes` is a list of  `__typeof_size`. 

For me, it is more clear to use `__typeof_sizes` to intend what this variable is.

Reply:
This is a list collection of `typeof(size_t)`. Therefore, "types of (sizes on all devices)" is more accurate, "a type of sizes" is wrong.

By the way, `typessof` is typo...

Reply:
OK. I understand your intention. 

I also checked typo is fixed